testing/features/automated_release_process.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Automated Release Process
+
+This is to automate the design system removing manual developer actions. 
+    
+As a design system developer 
+I need to be able to specify that a design system release is Alpha or Public
+So that users understand that the design system is ready for either the public or develop environments
```

Comment:
- who are the users of the design system / do we need to specify here

Reply:
For the purposes of the code part (the component library) our main users are other product teams in the organisation. e.g. Content platform, casebook, digital workplace and energy teams mainly. But in theory any product team. 

But there's also users in the sense of _software_ that depends on the design system code.

Reply:
Not sure if we want to cover the wider design system community here or if this should be technology-specific.

Reply:
I think changing `public or develop` to `production or development` is probably more accurate but I am splitting hairs.

Reply:
Thanks @davidsauntson - have changed to "production or development" - gotta split those hairs!

@davidrapson The way I understood it for this one it would be the product teams needing to be notified of changes.

In terms of the way other software (as users) depends on the code, would automating the release process change anything here in how it interacts with the design system - ie is there a requirement from a "user = software" perspective? I'd assume not, but am still getting to grips with the different pieces of the puzzle!